CMakeLists.txt

```diff
@@ -421,6 +421,7 @@ endif()
 
 add_custom_target(generate-message-map
    COMMAND vcpkg x-generate-default-message-map locales/messages.json
+   COMMAND vcpkg x-generate-default-message-map --no-output-comments locales/messages.en.json
```

Comment:
This fix is backwards and the ci command should be removed. (Note that messages.en.json is no longer ever used by the product code:

https://github.com/microsoft/vcpkg-tool/blob/d5f4ca6aaecf64b29079bfb12f8d5f99f189e3ab/src/vcpkg/base/messages.cpp#L266-L267

)